### PR TITLE
Add icons for the device_class 'update'

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

This will add `mdi:update` as fixed icon for device class `DEVICE_CLASS_UPDATE`

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
  - home-assistant/core#53945
  - home-assistant/architecture#598
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/common/const.ts
Comment (ludeeus) [inline src/common/const.ts:71]: I'm not too sure about this one, while the name matches, that icon does not make me think of updates, more of "jump forward" or something :man_shrugging: 

`mdi:arrow-up-bold-circle` is what we have used in the supervisor panel for this.

Comment (ludeeus) [reply src/common/const.ts:71]: personally, I think I prefer `mdi:package-up` over both those

Comment (mib1185) [reply src/common/const.ts:71]: yeah for sure, since `mdi:arrow-up-bold-circle` is already used, I will change to it, already because of an unified look and feel 👍

Comment (ludeeus) [reply src/common/const.ts:71]: The supervisor can change as well :wink:

Comment (mib1185) [reply src/common/const.ts]: OK, than ... what about `mdi:cellphone-system-update` ?
image

Comment (ludeeus) [reply src/common/const.ts]: That's looking more of an indication of downloading something

Comment (mib1185) [reply src/common/const.ts]: I'm not really an UX expert, but for me all three of them would be ok in context of an "update available" sensor  🙈

Comment (ludeeus) [reply src/common/const.ts]: How about `mdi:checkbox-blank-circle` when no update, and `mdi:arrow-up-bold-circle` when there is one?

Comment (matthiasdebaat) [reply src/common/const.ts]: I like `mdi:package-up` the most. It a good visualization and we may need to use `mdi:arrow-up-bold-circle` for something else in the future. Also credits for @ludeeus for suggesting `mdi:package` when there's no update.

Comment (ludeeus) [reply src/common/const.ts]: The switching icon logic is handled in `src/common/entity/binary_sensor_icon.ts`

Comment (mib1185) [reply src/common/const.ts]: thx for the input, will have a look the upcoming days 👍

Comment (ludeeus) [reply src/common/const.ts]: with_update
without_update

**Review by ludeeus — APPROVED** [review]
:heavy_check_mark:  Can be merged once the core PR has been merged

**Pull request merged**
